graphics/constants.py

```diff
@@ -11,7 +11,7 @@
 
 
 PLAYERS_COLORS = [
-    (128, 0, 32),
+    (0, 0, 0),
```

Comment:
Is black a nice color for player?

Reply:
I just used it for test, because I had difficulty to see who is who XD. Will change that